aiida_optimade/main.py

```diff
@@ -63,8 +63,8 @@
 if CONFIG.debug:
     LOGGER.info("DEBUG MODE")
 
-DOCS_ENDPOINT_PREFIX = f"{get_custom_base_url_path()}{BASE_URL_PREFIXES['major']}"
 APP = FastAPI(
+    root_path=CONFIG.root_path,
```

Comment:
I think we'd remove this it should read from `CONFIG` or we also explicitly set `base_url`.